### PR TITLE
Akka 2.6

### PR DESCRIPTION
**Pull request by johanandren**
Also Scala to 2.13

Refs #27 while at it

### CODE REVIEW

File: src/main/g8/default.properties [hunk 1]
```diff
@@ -1,5 +1,4 @@
 name = akka-quickstart-java
 description = Akka is a toolkit and runtime for building highly concurrent, distributed, and fault tolerant event-driven apps. This simple application will get you started building Actor based systems with Java.
-scala_version=2.12.8
-# TODO: update to stable once 2.6 has been released
-akka_version=2.6.0-RC1
+scala_version=2.13.1
+akka_version=maven(com.typesafe.akka, akka-actor_2.13, stable)
```
Comment (patriknw) [inline src/main/g8/default.properties]: shouldn't it be akka-actor-typed?
and there is a reference to _2.12 in pom.xml

Comment (raboof) [reply src/main/g8/default.properties]: either should be OK, until we stop publishing `akka-actor_2.13` artifacts ;)

Comment (patriknw) [reply src/main/g8/default.properties]: I see, this is only for fetching the latest version

**Comment by patriknw** [comment]
pls add logback.xml and then it can be merged

**Comment by johanandren** [comment]
Dang I messed up with that log conf commit and pushed it to master, trying to undo that mess now

**Comment by patriknw** [comment]
master isn't used, so no worries (we should remove that branch)

**Comment by johanandren** [comment]
Didn't even get that right, pushed to 2.6.x by mistake, continuing in #35

**Pull request closed without merging**